requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy==1.11.0
 networkx==1.11
--e git+https://github.com/pogodevorg/pgoapi.git/@3a02e7416f6924b1bbcbcdde60c10bd247ba8e11#egg=pgoapi
+-e git+https://github.com/pogodevorg/pgoapi.git#egg=pgoapi
```

Comment:
I think you need the add the tag? To get the latest update from a branch? This is what I put on my `requirements.txt`:

```
-e git+https://github.com/pogodevorg/pgoapi.git/@v0.43.3-alpha#egg=pgoapi
```